frontend/.idea/frontend.iml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module type="WEB_MODULE" version="4">
+<module type="JAVA_MODULE" version="4">
```

Comment:
Not sure about this change, was automatically done by IntelliJ

Reply:
It should stay a WEB_MODULE

Reply:
Will revert

Reply:
If intellij keeps changing this then something is weird. Which version are you using?

Reply:
IntelliJ? The latest version, 2021.2.3. The `runConfiguration` file is also added all the time...